core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -133,8 +153,12 @@ oppia.factory('threadDataService', [
       if (commitMsg) {
         payload.commit_message = commitMsg;
       }
-      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(
-        onSuccess).error(onFailure);
+      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(function() {
```

Comment:
I've decreased the open threads count below. Did you miss it ? Or are you referring to something else ?
Anyways I'll be moving that out from the success callback and putting it above(and reverting it in case of an error).